stream-chat-android-ui-components-sample/src/release/kotlin/io/getstream/chat/ui/sample/application/AppConfig.kt

```diff
@@ -0,0 +1,36 @@
+package io.getstream.chat.ui.sample.application
+
+import io.getstream.chat.ui.sample.data.user.User
+
+object AppConfig {
```

Comment:
This file is identical in debug and release, why do we have it duplicated?

Reply:
I have no idea TBH, do we have some kind of prod environment for sample app? Might be worth to have it with a cron job which occasionally purges all the data populates the db with some neat channels/messages.